src/tutorial/modal.component.tsx

```diff
@@ -2,12 +2,27 @@ import React from 'react';
 import { useTranslation } from 'react-i18next';
 import { Modal } from '@carbon/react';
 import { useConfig } from '@openmrs/esm-framework';
+import { useOnboarding } from '../onboarding-context';
+
 import styles from './styles.scss';
 
 const TutorialModal = ({ open, onClose }) => {
   const { t } = useTranslation();
   const config = useConfig();
   const tutorialData = config.tutorialData;
+  const { startOnboarding } = useOnboarding();
+
+  const handleWalkthroughClick = (tutorial) => {
+    const steps = [
+      {
+        target: '[aria-label="OpenMRS"]',
+        content: 'This is the OpenMRS logo. Click here to go back to the home page.',
+        disableBeacon: true,
+      },
+    ];
```

Comment:
Let's move the steps to the config file itself. For now, we can just add one step for each walkthrough, displaying the name of the walkthrough.

Reply:
@Piumal1999 
May be we can have like the content of the current walkthrough to the name of the walkthrough that's been clicked for now?
Or do we need to have another PR for it?

Reply:
> May be we can have like the content of the current walkthrough to the name of the walkthrough that's been clicked for now?

yeah. I think we can do it within this PR. (Demonstrating that _it loads the respective steps of the walkthrough when triggered_ would be enough for this PR )